docs/en/intro-to-loom.md

```diff
@@ -21,166 +21,3 @@ So, the Loom SDK allows for developers to spin up an unlimited number of sidecha
 But PlasmaChain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
 
 Thus, our company’s primary objectives over the next year are all related to growing PlasmaChain to become the “universal Layer 2” for high performance dapps.
-
-In the next chapter, you will learn how to deploy your first app to Loom PlasmaChain. Let's get started!
```

Comment:
Shouldn’t it still lead into the next article ?

Reply:
> Shouldn’t it still lead into the next article ?

I wasn't sure if it's okay to refer to the "Join Testnets" page as the next chapter.  But probably it's just fine and it makes a nice transition. I'll bring it back.